pom.xml

```diff
@@ -0,0 +1,88 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.github.giis-uniovi</groupId>
+    <artifactId>retorch-llm-rp</artifactId>
+    <version>1.0.0</version>
+    <packaging>pom</packaging>
+
+    <name>retorch-llm-rp</name>
+    <description>Replication packages of RETORCH combined with LLMs
+    </description>
+    <url>https://github.com/giis-uniovi/retorch-llm-rp</url>
+    <organization>
+        <name>Software Engineering Research Group (GIIS) - University of Oviedo, ES</name>
+        <url>http://giis.uniovi.es/</url>
+    </organization>
+
+    <properties>
+        <java.version>21</java.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <modules>
+        <module>retorch-llm-rp </module>
+    </modules>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-api</artifactId>
+                <version>2.0.16</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-source-plugin</artifactId>
```

Comment:
You do not need this to generate a jar that will not be published. Moreover, you do not need to generate a jar

Reply:
I've completely lost the focus, the pom now is minimized to:
- Only the artifact, group, etc
- The dependencies required
- Our metadata
- Some properties like the maven compiler and target version and the encoding
- License
What can be simplified or removed without impacting the stability of the package manager? @javiertuya

Reply:
Sorry! I've realized now about the <packaging> annotation, removed and pushed!